backend/local/readme.md

```diff
@@ -0,0 +1,11 @@
+start a local dynamo db
+
+```
+docker run -p 8000:8000 amazon/dynamodb-local
```

Comment:
I would at least recommend to add the `--rm` option.
But in the end, we should just encode this into a package.json script.

Reply:
I wonder if we can put this all into a single script that starts both the container and server doing some run-p. I think one issue is that the server is likely up much faster than the DB, so maybe the local server could poll for the DB to be up and connect once possible?
Another potential improvement for lokal development would be to add nodemon to the mix to automatically restart the local backend on server changes so that you can do quick-feedback local development. But this could always be added later.